web/README.md

```diff
@@ -25,22 +25,34 @@ $ cd oauth2l-web/web
 ### Docker
 Download [Docker](https://docs.docker.com/get-docker/) if not already installed. 
 
-Create the image by running the following command:
+#### Running the Web Application
+Create the images for the backend and the frontend by running the following command:
 ```bash
-$ docker build -t oauth2l-web .
+$ docker-compose up -d --build .
 ```

```

Comment:
Can this be run anywhere from oauth2l-web or do you have to be in the web directory?